anvil-rstudio-base/README.md

```diff
@@ -1,19 +1,82 @@
-# RStudio Docker Image
+| Current RStudio Docker Version | Docker Image URL | Date Updated | Questions or Feedback |
```

Comment:
How about `Latest image version`? I feel `RStudio Docker Version` is still a bit confusing, like is it talking about some other `RStudio Docker Version` maintained elsewhere or this `anvil-rstudio` image

Reply:
Sure! I can change to "Latest Image Version" or I can do "Latest Image Release". If you have a preference, let me know.

Reply:
Either way sounds good to me..thanks!